webdriver/tests/classic/new_session/unhandled_prompt_behavior.py

```diff
@@ -90,13 +90,13 @@ def test_unhandled_prompt_behavior_as_string(
     [
         (  # Check the default behavior with no handlers defined
             {},
-            "dismiss and notify",
+            {},
```

Comment:
> When the capability is given as string we should also serialize is as such to keep backward compatibility with existing clients that are not able to handle the object value yet. So isn't that then not a bug in the specification? @sadym-chromium and @jgraham

@whimboo I assume your concern is regarding this test. Please note that the test sends `{"unhandledPromptBehavior": handler}`, effectively `{"unhandledPromptBehavior": {}}`. So I guess the expected serialized capabilities are the same: `{"unhandledPromptBehavior": {}}`.

Reply:
The WebDriver spec [**serialize the user prompt handler**](https://w3c.github.io/webdriver/#dfn-serialize-the-user-prompt-handler) serilizes capability as string only if `fallbackDefault` is set. And `fallbackDefault` is set only if the capability is a string.

Reply:
I just noticed that we missed to implement these changes. We will do via https://bugzilla.mozilla.org/show_bug.cgi?id=1905889.